pkgs/development/python-modules/discordpy/default.nix

```diff
@@ -1,22 +1,27 @@
 { lib
-, fetchurl
-, buildPythonPackage
-, pythonOlder
+, python
 , withVoice ? true, libopus
-, asyncio
-, aiohttp
-, websockets
-, pynacl
 }:
 
 let
```

Comment:
The purpose of `python-packages.nix` is to provide a coherent set of packages that work together. We cannot have any overrides inside `python-packages.nix` because that may cause collisions. In this case we likely have to mark the package as broken.

Reply:
I am doing the same in #34188 which probably means it can only be packaged as an application.

Reply:
Correct.